file not found during make

I'm trying to build the latest version using vscode on windows 11 and I keep getting the below error. 
 CMake Error at man/cmake_install.cmake:164 (file):                                                                                                                               
    file INSTALL cannot find                                                                                                                                                       
    "C:/Users/Darren/libfido2/build/x64/dynamic/man/eddsa_pk_free.3.gz": No                                                                                                        
    error.                                                                                                                                                                         
  Call Stack (most recent call first):                                                                                                                                             
    cmake_install.cmake:41 (include)                                                                                                                                               
                                                                                                                                                                                    
C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Microsoft\VC\v170\Microsoft.CppCommon.targets(166,5): error MSB3073: The command "setlocal [C:\Users\Darren\libfid 
o2\build\x64\dynamic\install.vcxproj]                                                                                                                                              
C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Microsoft\VC\v170\Microsoft.CppCommon.targets(166,5): error MSB3073: "C:\Program Files\Microsoft Visual Studio\202 
2\Community\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe" -DBUILD_TYPE=Release -P cmake_install.cmake [C:\Users\Darren\libfido2\build\x64\dynamic\install.vcxp 
roj]                                                                                                                                                                               
C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Microsoft\VC\v170\Microsoft.CppCommon.targets(166,5): error MSB3073: if %errorlevel% neq 0 goto :cmEnd [C:\Users\D 
arren\libfido2\build\x64\dynamic\install.vcxproj]                                                                                                                                  
C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Microsoft\VC\v170\Microsoft.CppCommon.targets(166,5): error MSB3073: :cmEnd [C:\Users\Darren\libfido2\build\x64\dy 
namic\install.vcxproj]                                                                                                                                                             
C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Microsoft\VC\v170\Microsoft.CppCommon.targets(166,5): error MSB3073: endlocal & call :cmErrorLevel %errorlevel% &  
goto :cmDone [C:\Users\Darren\libfido2\build\x64\dynamic\install.vcxproj]                                                                                                          
C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Microsoft\VC\v170\Microsoft.CppCommon.targets(166,5): error MSB3073: :cmErrorLevel [C:\Users\Darren\libfido2\build 
\x64\dynamic\install.vcxproj]                                                                                                                                                      
C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Microsoft\VC\v170\Microsoft.CppCommon.targets(166,5): error MSB3073: exit /b %1 [C:\Users\Darren\libfido2\build\x6 
4\dynamic\install.vcxproj]                                                                                                                                                         
C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Microsoft\VC\v170\Microsoft.CppCommon.targets(166,5): error MSB3073: :cmDone [C:\Users\Darren\libfido2\build\x64\d 
ynamic\install.vcxproj]                                                                                                                                                            
C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Microsoft\VC\v170\Microsoft.CppCommon.targets(166,5): error MSB3073: if %errorlevel% neq 0 goto :VCEnd [C:\Users\D 
arren\libfido2\build\x64\dynamic\install.vcxproj]                                                                                                                                  
C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Microsoft\VC\v170\Microsoft.CppCommon.targets(166,5): error MSB3073: :VCEnd" exited with code 1. [C:\Users\Darren\ 
libfido2\build\x64\dynamic\install.vcxproj]                                                                                                                                        
Done Building Project "C:\Users\Darren\libfido2\build\x64\dynamic\install.vcxproj" (default targets) -- FAILED.                                                                    
 I looked in the folder and the size of that file is 0 bytes I was wondering if it was a typo or somethingthanks                                                                                                                                                                                  
 <!--

Please use the questions below as a template, and review your answers
for potentially sensitive information.

Thank you!

-->

**What version of libfido2 are you using?**

**What operating system are you running?**

**What application are you using in conjunction with libfido2?**

**How does the problem manifest itself?**

**Is the problem reproducible?**

**What are the steps that lead to the problem?**

**Does the problem happen with different authenticators?**

<!--

fido2-token is provided by the fido2-tools package on Debian and Ubuntu,
and shipped with libfido2 in macOS (Homebrew), Arch Linux, and Windows.

-->

**Please include the output of `fido2-token -L`.**

<details>
<summary><code>fido2-token -L</code></summary>
<br>
<pre>
$ fido2-token -L

</pre>
</details>

**Please include the output of `fido2-token -I`.**

<details>
<summary><code>fido2-token -I</code></summary>
<br>
<pre>
$ fido2-token -I &lt;device&gt;

</pre>
</details>

<!--

You are strongly encouraged to only capture debug output using test
credentials. Failure to do so can disclose sensitive information.

-->

**Please include the output of `FIDO_DEBUG=1`.**

<details>
<summary><code>FIDO_DEBUG=1</code></summary>
<br>
<pre>
$ export FIDO_DEBUG=1
$ &lt;command1&gt;
$ &lt;command2&gt;
(...)
$ &lt;commandn&gt;

</pre>
</details>

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -484,7 +484,7 @@ endif()
 if(BUILD_TOOLS)
 	add_subdirectory(tools)
 endif()
-if(BUILD_MANPAGES)
+if(BUILD_MANPAGES AND NOT MSVC)
 	add_subdirectory(man)
 endif()
 

--- a/man/CMakeLists.txt
+++ b/man/CMakeLists.txt
@@ -396,7 +396,7 @@ if(GZIP_PATH)
 		install(FILES ${PROJECT_BINARY_DIR}/man/${DST}.3.gz
 			DESTINATION "${CMAKE_INSTALL_MANDIR}/man3")
 	endforeach()
-elseif(NOT MSVC)
+else()
 	add_dependencies(man man_symlink)
 	foreach(f ${MAN_SOURCES})
 		if (${f} MATCHES ".1$")

